tests/integration_tests/performance/test_versioned_serialization_benchmark.py

```diff
@@ -68,8 +68,8 @@ def _check_statistics(directory, mean):
         attribute = "no-crc"
 
     measure = BASELINES[proc_model[0]][bench][attribute]
-    low = measure["target"] - measure["delta"]
-    high = measure["target"] + measure["delta"]
+    low = round(measure["target"] - measure["delta"], 3)
+    high = round(measure["target"] + measure["delta"], 3)
```

Comment:
`round` in Python is different to other languages. Are we sure we want `round` and not `math.trunc` (round towards 0)